h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -262,6 +264,29 @@ public boolean onExceptionalCompletion(Throwable ex, CountedCompleter caller) {
     }
 
     public abstract void computeImpl();
+    
+    public void computeEffectiveParameters() {
+      M model = _result.get();
+      if (model != null) {
+        model._effective_parms = (P) _parms.clone();
```

Comment:
is this necessary, clone() is called in model constructor

Reply:
its not, removed